From 441ea0fffb6d4af12b15545f72e159c84256fa53 Mon Sep 17 00:00:00 2001
From: Matthew Feickert <matthew.feickert@cern.ch>
Date: Mon, 23 Dec 2024 00:08:17 -0700
Subject: [PATCH] fix: Allow for longer paths for ffopen (#8)

* The ffopen subroutine path and fullname variables need to have enough
  characters to hold longer paths that can come up, such as the $PREFIX
  variable in conda-build builds, so extend the length to 512 (2^9) characters.
---
 src/ff/ffinit.f | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ff/ffinit.f b/src/ff/ffinit.f
index 463bd3b..7602e3c 100644
--- a/src/ff/ffinit.f
+++ b/src/ff/ffinit.f
@@ -756,7 +756,7 @@
 	character*(*) name
 *
 	logical lopen
-	character*128 path,fullname
+	character*512 path,fullname
 *
 	include 'ff.h'
 *
-- 
2.47.1

